index.html

```diff
@@ -80,21 +80,97 @@ <h4 class="list-heading">Skills</h4>
+                has been the industry's standard dummy text ever since the 1500s, when an unknown
+                printer took a standard dummy text.</p>
+            <ul class="programs">
+                <li class="lang">css &nbsp;|</li>
+                <li class="lang">html &nbsp;|</li>
+                <li class="lang">bootstrap |</li>
+                <li class="lang"> Ruby</li>
+            </ul>
+            <div>
+                <button class="button" type="button" onclick="">See Project</button>
+            </div>
+        </section>
+
+        <div class="middle-section">
+            <hr>
+            <img src="images/snapshot.png" alt="Snapshoot">
+            <h2>Multi-Post Stories </h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+                has been the industry's standard dummy text ever since the 1500s, when an unknown
+                printer took a standard dummy text.</p>
+            <div class="programs">
+                <div class="lang">css &nbsp;|</div>
+                <div class="lang">html &nbsp;|</div>
+                <div class="lang">bootstrap |</div>
+                <div class="lang"> Ruby</div>
+            </div>
+            <div>
+                <button class="button" type="button" onclick="">See Project</button>
+            </div>
+        </div>
+
+        <div class="middle-section">
+
+            <hr>
+            <img src="images/snapshot.png" alt="Snapshoot">
+            <h2>Multi-Post Stories </h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+                has been the industry's standard dummy text ever since the 1500s, when an unknown
+                printer took a standard dummy text.</p>
+            <div class="programs">
+                <div class="lang">css &nbsp;|</div>
+                <div class="lang">html &nbsp;|</div>
+                <div class="lang">bootstrap |</div>
+                <div class="lang"> Ruby</div>
+            </div>
+            <div>
+                <button class="button" type="button" onclick="">See Project</button>
+            </div>
+        </div>
+
+        <div class="middle-section">
+
+            <hr>
+            <img src="images/snapshot.png" alt="Snapshoot">
+            <h2>Multi-Post Stories </h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+                has been the industry's standard dummy text ever since the 1500s, when an unknown
+                printer took a standard dummy text.</p>
+            <div class="programs">
+                <div class="lang">css &nbsp;|</div>
+                <div class="lang">html &nbsp;|</div>
+                <div class="lang">bootstrap |</div>
+                <div class="lang"> Ruby</div>
+            </div>
+            <div>
```

Comment:
- [ ] as the previous reviewer requested kindly change all the technologies of all of the cards to `<ul><li>` tags